### PR TITLE
bug - adding indicator polling before calculating severity

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-22241)

## Description
Check that the indicator exists in the system before calculating it's severity, added indicator polling before getting the indicator from XSOAR

## Screenshots

Screen Shot 2023-03-20 at 17 28 13

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by GuyAfik — DISMISSED** [review]
nice

File: Packs/CommonPlaybooks/Playbooks/playbook-Calculate_Severity_By_Highest_DBotScore.yml [hunk 1]
```diff
@@ -404,10 +493,19 @@ inputs:
...
\ No newline at end of file
+- Calculate Severity - Generic v2 - Test
+- Wait Until Datetime - Test
+- URL Enrichment - Generic v2 - Test
+- File Enrichment - Generic v2 - Test
+- Domain Enrichment - Generic v2 - Test
+- Account Enrichment - Generic v2.1 - Test
+- Detonate URL - Generic Test
+- Email Address Enrichment - Generic v2.1 - Test
+- Endpoint Enrichment - Generic v2.1 - Test
```
Comment (GuyAfik) [inline Packs/CommonPlaybooks/Playbooks/playbook-Calculate_Severity_By_Highest_DBotScore.yml:510]: what are all of these test playbooks that you added?

**Review by GuyAfik — APPROVED** [review]
great!

File: Packs/CommonPlaybooks/Playbooks/playbook-Calculate_Severity_By_Highest_DBotScore.yml (hunk 1 above)
Comment (MLainer1) [reply Packs/CommonPlaybooks/Playbooks/playbook-Calculate_Severity_By_Highest_DBotScore.yml]: `demisto-sdk format`

**Review by idovandijk — COMMENTED** [review]
Looks good! Please see my comments:
1. Due to the version mismatches between the playbooks, please create a separate Calculate Severity playbook with a `fromversion` that is the same of the indicator registration polling playbook. Then add `toversion` to the existing Calculate Severity playbook.
2. Please test the flow with multiple indicators - try 20+ indicators. I'd like to make sure that it works without having to add a playbook loop.

**Review by idovandijk — APPROVED** [review]
Great!

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_54.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Indicator Registration Polling - Generic
+- Added the playbook to the xpanse marketplace.
```
Comment (dantavori) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_54.md:5]:
```suggestion
<!-- 
##### Indicator Registration Polling - Generic
- Added the playbook to the xpanse marketplace.
-->
```
Ideally we'd want this to appear only in XPANSE RNs but the <~XPANSE> tag isn't supported yet. So better just to hide this RN

**Pull request merged**